randovania/games/am2r/exporter/game_exporter.py

```diff
@@ -42,8 +42,14 @@ def export_can_be_aborted(self) -> bool:
     def _do_export_game(self, patch_data: dict, export_params: AM2RGameExportParams,
                         progress_update: status_update_lib.ProgressUpdateCallable):
         # Check if dotnet is available
-        dotnet_process = subprocess.run(["dotnet", "--info"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
-        if dotnet_process.returncode != 0:
+        dotnet_ran_fine = False
+        try:
+            dotnet_process = subprocess.run(["dotnet", "--info"],
+                                            stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=False)
+            dotnet_ran_fine = dotnet_process.returncode == 0
+        except FileNotFoundError:
+            dotnet_ran_fine = False
```

Comment:
Can you add a test case? if you do `mocker.patch("subprocess.run", side_effect=FileNotFoundError)` it'd work fine

Reply:
Test added